### ISSUE TITLE
OPAM installation fails 

### ISSUE THREAD

**Issue by Chris00** [inria-parkas/sundialsml#5]
Installing the package through OPAM fails with the following error:
```
#=== ERROR while installing sundialsml.2.7.0p0 ================================#
# opam-version 1.2.2
# os           linux
# command      make install-findlib
# path         /home/trch/.opam/4.04.0/build/sundialsml.2.7.0p0
# compiler     4.04.0
# exit-code    2
# env-file     /home/trch/.opam/4.04.0/build/sundialsml.2.7.0p0/sundialsml-20911-d2c37b.env
# stdout-file  /home/trch/.opam/4.04.0/build/sundialsml.2.7.0p0/sundialsml-20911-d2c37b.out
# stderr-file  /home/trch/.opam/4.04.0/build/sundialsml.2.7.0p0/sundialsml-20911-d2c37b.err
### stdout ###
# ocamlmklib -ocamlc "ocamlc.opt -annot " -ocamlopt "ocamlopt.opt " -verbose	\
# [...]
#     -lsundials_cvodes		\
#     		\
#     -lsundials_idas		\
#     -lsundials_kinsol		\
#     -lsundials_nvecserial -lm -llapack -lblas
# + gcc -shared  -o ./dllmlsundials.so sundials/sundials_ml.o lsolvers/dls_ml.o nvectors/nvector_ml.o lsolvers/spils_ml.o kinsol/kinsol_ml.o cvodes/cvode_ml_s.o cvodes/cvodes_ml.o cvodes/cvode_klu_ml_s.o cvodes/cvodes_klu_ml.o cvodes/cvode_superlumt_ml_s.o cvodes/cvodes_superlumt_ml.o idas/ida_ml_s.o idas/idas_ml.o idas/ida_klu_ml_s.o idas/idas_klu_ml.o idas/ida_superlumt_ml_s.o idas/idas_superlumt_ml.o arkode/arkode_klu_ml.o arkode/arkode_superlumt_ml.o kinsol/kinsol_klu_ml.o kinsol/kinsol_superlumt_ml.o    -lsundials_cvodes -lsundials_idas -lsundials_kinsol -lsundials_nvecserial -lm -llapack -lblas
# Makefile:17: recipe for target 'sundials.cma' failed
# make[1]: Leaving directory '/home/trch/.opam/4.04.0/build/sundialsml.2.7.0p0/src'
# Makefile:57: recipe for target 'install-findlib' failed
### stderr ###
# /usr/bin/ld: /usr/lib/gcc/x86_64-linux-gnu/7/../../../../lib/libsundials_cvodes.a(cvodes.o): relocation R_X86_64_PC32 against symbol `cvSensRhs1InternalDQ' can not be used when making a shared object; recompile with -fPIC
# /usr/bin/ld: final link failed: Bad value
# collect2: error: ld returned 1 exit status
# make[1]: *** [sundials.cma] Error 2
# make: *** [install-findlib] Error 2
```

**Comment by tbrk** [comment]
Which distribution are you installing on?
How did you install the underlying Sundials library?
Thanks.

**Comment by Chris00** [comment]
I am installing on Debian testing.  I had `libsundials-serial-dev` installed at the time of the report — there is a problem between libssl and isync that held that package.  I changed to `libsundials-dev`  (the former actually no longer exists in Debian/testing) and the opam package now installs correctly.

**Comment by tbrk** [comment]
It should definitely work on Debian (I use stable on one of my machines). Thanks for the tip about `libsundials-dev`. I will update the opam package soon (I leave this issue open to remind me). Please let me know if you have any more problems or questions. Everything except for the parallel features pending ocaml/opam-repository#5023.

**Comment by tbrk** [comment]
I've just had a closer look at this issue as part of the preparation for the next release. To support both debian-stable and debian-testing, it seems that the new opam features proposed in ocaml/opam#2919  would be needed. This issue may thus have to stay open for a bit longer.

**Comment by tbrk** [comment]
I've finally got around to updating the `conf-sundials` package in opam which fixes this build problem. I've also released Sundials/ML 5.8.0p0 which supports all versions of Sundials from 2.7.0 through 5.8.0, so the installation should now work smoothly again after an `opam update`.
